src/Sylius/Bundle/CoreBundle/Fixture/CatalogPromotionRuleFixture.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Fixture;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+final class CatalogPromotionRuleFixture extends AbstractResourceFixture
```

Comment:
IMO this fixture file is probably not needed and this configuration tree should be placed in CatalogPromotionFixture